build/docker/Dockerfile-bare

```diff
@@ -0,0 +1,41 @@
+# This dockerfile builds the zap stable release
+FROM ubuntu:16.04 as builder
+MAINTAINER Simon Bennetts "psiinon@gmail.com"
+
+WORKDIR /zap
+
+RUN apt-get update && apt-get install -q -y --fix-missing \
+	wget \
+	curl \
+	xmlstarlet \
+	unzip && \
+	apt-get clean && \
+	rm -rf /var/lib/apt/lists/*
+
+# Download and expand the latest stable release 
+RUN curl -s https://raw.githubusercontent.com/zaproxy/zap-admin/master/ZapVersions-dev.xml | xmlstarlet sel -t -v //url |grep -i Linux | wget --content-disposition -i - -O - | tar zxv && \
+	cp -R ZAP*/* . &&  \
+	rm -R ZAP* 
+
+FROM java:openjdk-8-jdk-alpine
+MAINTAINER Simon Bennetts "psiinon@gmail.com"
+
+WORKDIR /zap
+COPY --from=builder /zap .
+RUN echo "http://dl-3.alpinelinux.org/alpine/edge/main" >> /etc/apk/repositories &&\
+    apk add --update --no-cache bash netcat-openbsd && \
```

Comment:
This spacing could use tabs (to normalise with other lines in this and other Docker files).

Reply:
just checked, and it is tab instead of space - unless I am missing something

Reply:
It's using spaces here, in a following line is using a tab (which stands out from the surrounding lines).

Reply:
ok, I tried to clean it up a bit, hope that solved it

Reply:
Looks good. Could you squash the commits?

Reply:
I can, but usually, this is done through GitHub - if this is not an option let me know and I'll squash them.

Reply:
We usually squash before merge, but I'm OK with squash while merging the pull request as well.